plugins/providers/client.go

```diff
@@ -10,6 +14,7 @@ type Client interface {
 	RevokeAccess(*domain.ProviderConfig, domain.Grant) error
 	GetRoles(pc *domain.ProviderConfig, resourceType string) ([]*domain.Role, error)
 	GetAccountTypes() []string
+	ListAccess(context.Context, domain.ProviderConfig) (domain.ResourceAccess, error)
```

Comment:
@rahmatrhd As we were discussing, should this provider accept list of resources for which to return access list?